web_monitoring/html_diff_render.py

```diff
@@ -810,17 +926,51 @@ def html(self):
 # I had some weird concern that I needed to make this token a single word with
 # no spaces, but now that I know this differ more deeply, this is pointless.
 class ImgTagToken(tag_token):
-    def __new__(cls, tag, data, html_repr, pre_tags=None,
+
+    regex = re.compile(r"(src|srcset)=('|\")(.*?)('|\")")
+
+    def are_any_urls_equal(self, other):
+        strict_urls = self.strict_urls
+        self_array = self[7:-3]
```

Comment:
Once we get to this point, we should hopefully not be doing any more parsing of strings. I think you should probably be modifying `flatten_el`, `fixup_chunks`, and `_customize_token` here. See below for more.